src/lib/variants.ts

```diff
@@ -219,7 +219,10 @@ export const withinMaxViews = (log: ViewLog, now: Date = new Date()): Filter =>
     },
 });
 
-export const withinArticleViewedSettings = (history: WeeklyArticleHistory): Filter => ({
+export const withinArticleViewedSettings = (
+    history: WeeklyArticleHistory,
+    now: Date = new Date(),
+): Filter => ({
     id: 'withinArticleViewedSettings',
```

Comment:
Same as in previous comment. Not necessarily related to this PR but needed so these tests don't start failing as the  test data becomes stale.